pkg/cli/env.go

```diff
@@ -46,6 +46,7 @@ func init() {
 			flagId,
 			flagRack,
 			flagWait,
+			flagReplace,
```

Comment:
Sorry one more thing. The `flag*` globals are really only meant for variables that are used in more than one command. Could you please change this to look more like the promote flag below (and alphabetize below it)